translations/i18nCheck.R

```diff
@@ -0,0 +1,66 @@
+# For developers to detect unreasonable calls in code to translate strings.
+# both applies to the R and QML code of the analysis module.
+
+library(potools)
```

Comment:
This file works fine, but I do have a few stylistic comments that I'll leave as suggestions below.